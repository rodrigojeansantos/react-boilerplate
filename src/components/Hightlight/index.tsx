import * as S from './styles'

const Hightlight = () => (
  <S.Wrapper>
    <h1>Hightlight</h1>
  </S.Wrapper>
)

export default Hightlight
