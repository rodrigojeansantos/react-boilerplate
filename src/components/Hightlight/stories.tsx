import { Story, Meta } from '@storybook/react'
import Hightlight from '.'

export default {
  title: 'Hightlight',
  component: Hightlight
} as Meta

export const Default: Story = () => <Hightlight />
