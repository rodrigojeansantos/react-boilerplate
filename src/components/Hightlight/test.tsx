import { render, screen } from '@testing-library/react'

import Hightlight from '.'

describe('<Hightlight />', () => {
  it('should render the heading', () => {
    const { container } = render(<Hightlight />)

    expect(
      screen.getByRole('heading', { name: /Hightlight/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
