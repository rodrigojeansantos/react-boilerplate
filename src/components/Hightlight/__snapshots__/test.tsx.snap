// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hightlight /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Hightlight
  </h1>
</main>
`
